src/components/StorageOverview/TopConsumers/tests/TopConsumers.test.js

```diff
@@ -9,10 +9,6 @@ describe('<TopConsumers />', () => {
     const component = render(<TopConsumers {...TopConsumerStats[0]} />);
     expect(component).toMatchSnapshot();
   });
-  it('data-renders-correctly', () => {
```

Comment:
Why is this test being removed?

Reply:
At [line](https://github.com/kubevirt/web-ui-components/blob/657b7a4a13d563ba9f8be5891623112e74539e43/src/selectors/prometheus/storage.js#L31), new Date() method has been used, which converts the epoch timestamp to the local the time zone. When `yarn jest -u` command runs on my system, the snapshot that gets created, has the date as per my timezone. In the Travis CI environment, date that gets calculated of the same timestamp(passed in the fixture) is different as timezone differs. Hence, the `yarn test` fails. You can refer this build [log](https://travis-ci.com/kubevirt/web-ui-components/builds/108665012). 
Hence, thought of removing it. I feel that I can comment the unit test and add the description there for its explanation.

Reply:
Found one solution to this - https://jestjs.io/docs/en/snapshot-testing#property-matchers. Will try it out.

Reply:
Or we can mock the Data or transformation, so it will be reliable for tests